ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Cli.scala

```diff
@@ -146,6 +146,12 @@ object Cli {
       .text("""A comma-separated list of inclusion prefixes. If not specified, all default tests are included. If specified, only tests that match at least one of the given inclusion prefixes (and none of the given exclusion prefixes) will be run. Can be specified multiple times, i.e. `--include=a,b` is the same as `--include=a --include=b`.""")
 
     opt[Seq[String]]("perf-tests")
+      .validate(tests =>
+        if (tests.forall(Tests.PerformanceTestsKeySet)) {
+          success
+        } else {
+          failure("Invalid performance test name, use --list to see valid performance test names")
+      })
```

Comment:
Or perhaps something like this:

```suggestion
      .validate(tests =>
        tests.find(Tests.PerformanceTestsKeySet) match {
          None =>
            success
          Some(test) =>
            failure(s"Invalid performance test name: '$test'. Use `--list` to see valid names.")
      })
```

Reply:
I'll use this suggestion, but I believe this logic is the opposite of what we want. :wink:

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6826/commits/ed7c37fb0a12564901f8bbdf63d080b35a523d49
Thanks for the suggestion. :bow:

Reply:
Eh, you figured it out. 😛